infra/terraform/hash/hash_application/temporal_worker_ai_py.tf

```diff
@@ -0,0 +1,56 @@
+locals {
+  temporal_worker_ai_py_service_name   = "worker-ai-py"
+  temporal_worker_ai_py_prefix         = "${var.prefix}-${local.temporal_worker_ai_py_service_name}"
+  temporal_worker_ai_py_param_prefix   = "${local.param_prefix}/worker/ai/py"
+}
+
+resource "aws_ssm_parameter" "temporal_worker_ai_py_env_vars" {
+  # Only put secrets into SSM
+  for_each = { for env_var in var.temporal_worker_ai_py_env_vars : env_var.name => env_var if env_var.secret }
+
+  name = "${local.temporal_worker_ai_py_param_prefix}/${each.value.name}"
+  # Still supports non-secret values
+  type      = each.value.secret ? "SecureString" : "String"
+  value     = each.value.secret ? sensitive(each.value.value) : each.value.value
+  overwrite = true
+  tags      = {}
+}
+
+locals {
+  temporal_worker_ai_py_service_container_def = {
+    essential = true
+    name      = local.temporal_worker_ai_py_prefix
+    image     = "${var.temporal_worker_ai_py_image.url}:latest"
+    cpu       = 0 # let ECS divvy up the available CPU
+    dependsOn   = [{ condition = "HEALTHY", containerName = local.temporal_worker_ai_ts_service_container_def.name }]
```

Comment:
Why does the Python worker depend on the TypeScript worker?

Reply:
I marked it as a dependency as the Python workflows use Typescript workflows but I guess this is solved by temporal itself anyway.
```suggestion
```